vue-frontend/tailwind.config.cjs

```diff
@@ -61,9 +61,11 @@ module.exports = {
         "product-sans": ["Product Sans"],
         "futura-bold": ["FuturaLT-Bold"],
         "roboto-bold": ["Roboto-Bold"],
+        "roboto-medium": ["Roboto-Medium"],
         "inter-bold": ["Inter-Bold"],
-        "inter-semibold": ["Inter-Semibold"],
         "inter-medium": ["Inter-Medium"],
+        "inter-semibold": ["Inter-Semibold"],
```

Comment:
The font-family name should be the same as a tailwind.css font class.  Its `SemiBold` not `Semibold`